src/PowerShellEditorServices/Services/DebugAdapter/Debugging/VariableDetails.cs

```diff
@@ -27,7 +28,7 @@ internal class VariableDetails : VariableDetailsBase
         /// </summary>
         public const string DollarPrefix = "$";
 
-        private object valueObject;
+        protected object valueObject;
```

Comment:
If it's going to be used outside of the declaring class it's best to make this a property instead of a field. Preferably a `{ get; }` if you can init it in the ctor, otherwise `{ get; private set; }`

Reply:
Implemented 8179caa03cedbbf4600397808f87202ca23779af

Reply:
Hmm looks like it's still a field. You'd want this:

```csharp
protected object ValueObject { get; }
```

Fields are usually kept to private to the class implementation detail only. Properties make it easier to change how it's accessed later. Not a huge deal since it's all internal anyway but it can still make refactoring a pain later on.